.github/workflows/nightly_runner.yml

```diff
@@ -9,12 +9,12 @@ jobs:
     name: Run tests with Python ${{ matrix.python-version }} on ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [ '3.6', '3.7', '3.8', '3.9', '3.10' ]
+        python-version: [ '3.6', '3.7', '3.8', '3.9', '3.10', 'pypy-3.7', 'pypy-3.8', 'pypy-3.9']
```

Comment:
IMO it's adequate to check only the latest PyPy version.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-python-client/pull/571/commits/2877cd06a3efb7abd4ec357a3842a40c80b74edc.